.cirrus.yml

```diff
@@ -30,13 +31,21 @@ nix_test_task:
 task:
   auto_cancellation: true
 
-  env:
-    matrix:
-      PY_VER: "3.6"
-      PY_VER: "3.7"
+  matrix:
```

Comment:
could you refactor this out of the windows/(linux+osx) tasks into one place using YAML aliases?

Reply:
I can try

Reply:
I ended up matrixing it all